### ISSUE TITLE
VideoRecorderPlugin segfaults when there is no camera connected (Trac #22)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#26]
Migrated from http://microfluidics.utoronto.ca/ticket/22
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "", 
    "reporter": "cfobel", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "VideoRecorderPlugin segfaults when there is no camera connected", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-01-04T03:24:02", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120104T04:41:27: 
* christian@fobel.net commented:

> commit 8a211d3a4990c63e34a80df3ff146b31faff7213
> Author: Christian Fobel <christian@fobel.net>
> Date:   Tue Jan 3 23:28:05 2012 -0500
> 
>     Fail gracefully on exception during plugin init...
>     
>     Fail gracefully on exception during plugin initialization.  Fixes #26.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "core" to "microdrop"
